Sources/ResilientDecoding/ResilientArray+DecodingOutcome.swift

```diff
@@ -0,0 +1,98 @@
+// Created by George Leontiev on 4/19/20.
+// Copyright © 2020 Airbnb Inc. All rights reserved.
+
+import Foundation
+
+extension Resilient {
+  
+  init<T>(_ results: [Result<T, Error>]) where Value == [T] {
+    self.init(results, transform: { $0 })
+  }
+  
+  init<T>(_ results: [Result<T, Error>]) where Value == [T]? {
+    self.init(results, transform: { $0 })
+  }
+
+  private init<T>(_ results: [Result<T, Error>], transform: ([T]) -> Value) {
```

Comment:
Why have this transform parameter if we're always passing `{ $0 }`?

Reply:
Good catch. This is vestigial.

Reply:
Actually, I take that back. While they are spelt the same, they are different because of type inference. I'll add a comment.